src/app/api/oauth/callback/page.tsx

```
@@ -0,0 +1,80 @@
+
+        let provider = 'kakao';
+
+        const storedProvider = sessionStorage.getItem('oauth_provider');
+        if (storedProvider) {
+          provider = storedProvider;
+          sessionStorage.removeItem('oauth_provider');
+        }
+
+        console.log('OAuth 콜백 처리:', { code: code, provider });
+
+        const { response, headers, status } = await AuthService.oauthLogin(client, {
```

ESLint

[error] 'headers' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)